src/k-provider/ott/provider.js

```diff
@@ -128,6 +128,18 @@ export default class OTTProvider extends BaseProvider<OTTProviderMediaInfoObject
     return mediaConfig;
   }
 
+  /**
+   * Gets the backend playlist config.
+   * @returns {Promise<Error>} - Not implemented method
+   */
+  getPlaylistConfig(): Promise<Error> {
```

Comment:
this needs to be implemented in the base class and updated in the interface definition.
Also - what will be the result of throwing returning a rejected promise here in the kaltura-player?

Reply:
I don't think we need a rejected promise, success with empty response is also valid and if you think of the outcome it doesn't matter in high level as we still don't support playlists in OTT.

Reply:
1. kaltura-player catches the error and dispatches an error (like missing params flow)
2. we do support playlist in OTT by entry list